ferolino_jasmin/country.html

```diff
@@ -8,15 +8,15 @@
 </head>
 <body>
     <h1>Countries of the World</h1>
-    <input type="text" id="countryInput" placeholder="Enter a country">
+    <input type="text" id="country_input" placeholder="Enter a country">
     <button onclick="searchCountry()">Search</button>
 
-    <div id="countryDetails">
+    <div id="countrydetails">
     </div>
 
-    <div id="countriesInRegion">
+    <div id="countrieinregion">
     </div>
 
-    <script src="countries.js"></script>
+    <script src=" javascript.js"></script>

```

Comment:
### Remove unnecessary/excessive lines and spacing.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d18931cb7_3_0

No space in the script tag for the file name of your js.